build-system/eslint-rules/preact-preferred-props.js

```diff
@@ -51,6 +51,18 @@ module.exports = {
         }
         const property = prop.key.name || prop.key.value;
 
+        if (property?.toLowerCase() === 'tabindex') {
+          context.report({
+            node: prop,
+            message:
+              '`tabindex` and `tabIndex` should be supported equally.' +
+              ' Instead of destructuring, use:' +
+              `\n\timport {tabindexFromProps} from '#preact/utils';` +
+              `\n\t<... tabindex={tabindexFromProps(rest)}>`,
```

Comment:
I added `tabindexFromProps()` for backwards-compatibility in component consumers, since we apply the mapping both ways. 

The only component that takes `tabindex` is `Accordion`. We may determine that this user is okay to break, in which case we can remove the additional lint rules for the utility.